uploader.json

```diff
@@ -1070,14 +1162,45 @@
                 "ImagePathError: Image path does not exist or cannot be accessed/created:",
                 "Migration is stuck: Hasn't progressed in",
                 "The name org.fedoraproject.FirewallD1 was not provided by any .service files",
-                "Timeout while waiting for path preparation"
+                "Timeout while waiting for path preparation",
+                "Unable to obtain correct plug status or plug is not available"
             ]
         },
         {
             "file": "/etc/vdsm/vdsm.id",
             "pattern": []
         },
         {
+            "file": "/etc/()*virt-who\\.conf",
+            "pattern": [
+                "[",
+                "configs",
+                "debug",
+                "env",
+                "interval",
+                "log_",
+                "oneshot",
+                "owner",
+                "server",
+                "type"
+            ]
+        },
+        {
+            "file": "/etc/virt-who\\.d/()*.*\\.conf",
```

Comment:
Due to the way the uploader works, this will never be collected.  You can't put pattern expressions in directory paths.

Reply:
OK, will fix it.

Reply:
Please review: https://github.com/RedHatInsights/insights-core/pull/612